go/staking/api/api_test.go

```diff
@@ -506,3 +507,151 @@ func TestReclaimEscrowResultsSerialization(t *testing.T) {
 		require.EqualValues(tc.rr, dec, "ReclaimEscrow serialization should round-trip")
 	}
 }
+
+func TestEventsSerialization(t *testing.T) {
+	require := require.New(t)
+
+	pk1 := signature.NewPublicKey("aaafffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
+	addr1 := NewAddress(pk1)
+	pk2 := signature.NewPublicKey("bbbfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
+	addr2 := NewAddress(pk2)
+	var txHash hash.Hash
+	txHash.Empty()
+
+	// NOTE: These cases should be synced with tests in runtime/src/consensus/staking.rs.
```

Comment:
Long term, instead of hardcoding crap like this, we should start sticking testvectors into JSON files that are used by both test cases.